Arrangement_on_surface_2/include/CGAL/Surface_sweep_2/Arr_construction_ss_visitor.h

```diff
@@ -142,6 +142,9 @@ class Arr_construction_ss_visitor :
   /* A notification issued before the sweep process starts. */
   inline void before_sweep();
 
+  /*! A notification issued after the sweep process ends. */
+  inline void after_sweep();
```

Comment:
why is this needed?

Reply:
To call [`clean_inner_ccb()`](https://github.com/CGAL/cgal/pull/4851/files/1be446983801c00884d785ff9acc16fe5f913f75#diff-c697bc93f9992191b6ca431c9ee36d37R280).